src/main/java/ee/tuleva/onboarding/auth/smartid/SmartIdSession.java

```diff
@@ -24,4 +25,9 @@ public void setAuthenticationIdentity(AuthenticationIdentity identity) {
     lastName = identity.getSurname();
     country = identity.getCountry();
   }
+
+  @Override
+  public String getPhoneNumber() {
+    return null;
+  }
```

Comment:
We should not have the phoneNumber as part of the top-level interface if it's not relevant for all the implementation classes.